src/components/layout/nav/signup/SignUp.scss

```diff
@@ -0,0 +1,79 @@
+.signupBox {
+  width: 250px;
+  transition: all 1s ease-in-out;
+  position: absolute;
+  margin: 0 auto;
+  z-index: -1;
```

Comment:
why is there a z-index here?

btw in general we try to avoid using z-indexes unless we absolutely have to, i see a lot of z-indexes in your css and it's a bit fishy. you should use them when they're needed but also try to avoid it

https://danielrotter.at/2020/04/08/avoid-z-index-whenever-possible.html